flutter_application_1/windows/flutter/generated_plugins.cmake

```diff
@@ -3,6 +3,7 @@
 #
 
 list(APPEND FLUTTER_PLUGIN_LIST
+  permission_handler_windows
 )
```

Comment:
Remove the `windows` directory (for now) as we do not plan to publish / run this app on Windows